src/SourceBuild/content/build.proj

```diff
@@ -9,6 +9,8 @@
 
   <Import Project="Sdk.targets" Sdk="Microsoft.NET.Sdk" />
 
+  <UsingTask AssemblyFile="$(PublishVerticalManifestAssembly)" TaskName="PublishVerticalManifest" />
```

Comment:
After you renamed the task, please use full qualified name here. See https://github.com/dotnet/arcade/blob/cd27fcf3c2b5c238acd3e8d1e85bb6ef6c205686/src/Microsoft.DotNet.Arcade.Sdk/tools/GenerateChecksums.targets#L4 as an example.